src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVenmoAccount.java

```diff
@@ -48,16 +50,16 @@ public HyperwalletVenmoAccount clearToken() {
         return this;
     }
 
-    public HyperwalletTransferMethod.Type getType() {
+    public Type getType() {
         return type;
     }
 
-    public void setType(HyperwalletTransferMethod.Type type) {
+    public void setType(Type type) {
```

Comment:
if the type is for this class `HyperwalletVenmoAccount` is always `VENMO_ACCOUNT` why do we allow users of this model to modify it?